plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -17,71 +16,15 @@ import (
 	"strings"
 )
 
-// mask for masking username/password from error messages
-var mask = regexp.MustCompile(`https?:\/\/\S+:\S+@`)
```

Comment:
Ah - Cameron put in a change the same day which added this and its functionality. Presumably this would be fixed with a merge.